web/styles/zulip.css

```diff
@@ -838,6 +838,14 @@ strong {
             .typeahead-text-container {
                 align-self: center;
 
+                &.has_rendered_markdown {
+                    /* This makes the overall width of the typeahead to be 292px. */
+                    max-width: 270px;
```

Comment:
Should we just put 292px max-width on `a` tag wrapping this so that we can still use the extra space occupied by the `unsubscribed_icon` in the typeahead?

Reply:
OK, Removed `max-width` from here and added it to `<a>` tag.